element-templates/RESTConnectorTemplate.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://unpkg.com/@camunda/element-templates-json-schema/resources/schema.json",
-  "name": "REST Connector",
+  "name": "BP3 REST Connector",
```

Comment:
Should we rename the file too?

Reply:
Yes good idea, maybe `bp3-rest-connector.json`?